code/modules/client/preferences.dm

```diff
@@ -109,9 +109,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 		load_path(parent.ckey)
 		if(load_and_save && !fexists(path))
 			try_savefile_type_migration()
-		unlock_content = !!parent.IsByondMember()
+		unlock_content = !!parent.IsByondMember() || GLOB.donator_list[parent.ckey] //SKYRAT EDIT - ADDED DONATOR CHECK
```

Comment:
```suggestion
		unlock_content = parent.IsByondMember() || GLOB.donator_list[parent.ckey] //SKYRAT EDIT - ADDED DONATOR CHECK
```

Reply:
The double negative is there from tg. Probably to reduce it to 0 or 1, instead of what the IsByondMember() returns (-1 for permanent membership, or remaining days for non-permanent)

Reply:
oh nvm then